store/src/java/com/zimbra/cs/mailbox/MailSender.java

```diff
@@ -1478,4 +1479,31 @@ public static void unregisterPreSendMailListener(PreSendMailListener listener) {
             }
         }
     }
+
```

Comment:
Please add method documentation for both of the methods.

Reply:
done